README.md

```diff
@@ -56,10 +56,11 @@ version numbers.
 
 One of the following two options must be specified:
 
-* `regexp`: *Optional.* The pattern to match filenames against within S3. The first
-  grouped match is used to extract the version, or if a group is explicitly
-  named `version`, that group is used. At least one capture group must be
-  specified, with parentheses.
+* `regexp`: *Optional.* The forward-slash (`/`) delimited list of patterns to
```

Comment:
I see. I'll change `list` for `sequence`. Tell me if it's better that way:
```markdown
* `regexp`: *Optional.* The forward-slash (`/`) delimited sequence of patterns to
  match against the sub-directories and filenames of the objects stored within                                                                                 
  the S3 bucket. The first grouped match is used to extract the version, or if
  a group is explicitly named `version`, that group is used. At least one
  capture group must be specified, with parentheses.
```

Reply:
Put aside the fact that `regexp` cannot be changed to `path` without breaking existing pipelines, it might be a better approach to explain it.

I also found a realistic example overnight. What about this:
```markdown
* `regexp`: *Optional.* A POSIX-style path (components are separated by a forward slash `/`)
  to match against the sub-directories and filenames of the objects stored within the S3 bucket.
  Each path component is matched as a regular expression. The first grouped match is used to
  extract the version, or if a group is explicitly named `version`, that group is used. 
  At least one capture group must be specified, with parentheses.
  For example:
  - `finals/release-(\\d+\\.\\d+)/my-app-(?P<version>\\d+\\.\\d+\\.\\d+)\\.zip`
  - `artifacts/branch-foo/unit-test-(.*)\\.tgz`
```

Reply:
Yes, good improvement!